seqr/views/apis/individual_api.py

```diff
@@ -789,10 +789,10 @@ def save_individuals_metadata_table_handler(request, project_guid, upload_file_i
 def get_individual_rna_seq_data(request, individual_guid):
     individual = Individual.objects.get(guid=individual_guid)
     check_project_permissions(individual.family.project, request.user)
-    sample = Sample.objects.get(individual=individual, is_active=True, sample_type=Sample.SAMPLE_TYPE_RNA)
+    outlier_data = RnaSeqOutlier.objects.filter(sample__individual=individual, sample__is_active=True)
```

Comment:
The change is because the `Sample.objects.get()` fails for multiple matched objects. We could add tissue-type conditions, but it is not general as the new code.